crates/telio-model/src/api_config.rs

```diff
@@ -104,6 +104,22 @@ pub struct FeatureNurse {
     pub qos: Option<FeatureQoS>,
     /// Enable/disable collecting nat type
     pub enable_nat_type_collection: Option<bool>,
+    /// Enable/disable Relay connection data
+    #[serde(default = "FeatureNurse::get_default_enable_relay_conn_data")]
+    pub enable_relay_conn_data: bool,
```

Comment:
Why? It's either enabled or disabled, there is no third option...

Reply:
When I had added `enable_nat_type_collection` which was also an enable/disable, there was a problem when sending telio config (I don't recall the exact problem tbh) but because of which I had changed it to option<bool>.

Reply:
I feel bad for this kind of solution, because usually `Some(false) == None` which is kinda weird, but I can do it if that is needed.